docs/2.9.0/docs/canton/usermanual/upgrading.rst

```diff
@@ -476,35 +519,59 @@ side by side with the old sync domain process.
 
 This applies to all sync domain services, be it sequencer, mediator, or topology manager.
 
-Please note that currently, the sync domain ID cannot be preserved during upgrades.
-The new sync domain must have a different sync domain ID because the participant
-associates a sync domain connection with a sync domain ID, and that association
-must be unique.
+Wit that, the protocol upgrade process boils down to:
+
+#. **Deploy a new sync domain with the new protocol version**
+
+   Deploy a new sync domain next to the old sync domain and ensure that the new sync
+   domain is using the desired protocol version.
+
+   Also make sure to use different databases (or at least different schemas in the same database)
+   for the sync domain services (mediator, sequencer node, and topology manager), channel names,
+   smart contract addresses, etc.
+
+   The new sync domain must be completely separate, but you can reuse
+   your DLT backend as long as you use different sequencer contract addresses or Fabric channels.
+
+#. **Carry out a hard sync domain migration**
+
+   Instruct the participants individually using the :ref:`hard sync domain migration<hard_sync_domain_upgrade>`
+   to use the new sync domain.
+
+.. note::
+
+    Currently, the sync domain ID cannot be preserved during upgrades.
+    The new sync domain must have a different sync domain ID because the participant
+    associates a sync domain connection with a sync domain ID, and that association
+    must be unique.
```

Comment:
As usual, I would make how-to guides as short as possible. Consider removing the second sentence which answers to a WHY question.